client/src/views/components/button/button.jsx

```diff
@@ -48,7 +52,9 @@ export default class Button extends React.Component {
       className += " disabled";
     }
     if (this.props.destructive){
-      className += " destructive";
+      className += " destructive ghost";
```

Comment:
Is this intended to be ghost when props.ghost is not set?

Reply:
Yes. 

Ghost buttons are less distinctive than raised (filled) buttons. Raised buttons are sort of like the suggested or recommended course of action, while ghost buttons are secondary actions that have less prominence. Usually, having the safe option be the suggested action is safer than having the irreversible, potentially mistaken option be the suggested action.

Reply:
Sounds good to me